include/mxnet/lib_api.h

```diff
@@ -38,6 +38,154 @@
 
 #define MX_LIBRARY_VERSION 1
 
+/*
+ * Import from DLPack https://github.com/dmlc/dlpack/blob/master/include/dlpack/dlpack.h
```

Comment:
dlpack is 3rdparty of MXNet. Could we include <dlpack/dlpack.h> rather than copy its code?

Reply:
We want to keep everything in a single header file

Reply:
I see.